client/src/ui/BitbakeCommands.ts

```diff
@@ -47,6 +48,9 @@ export function registerBitbakeCommands (context: vscode.ExtensionContext, bitba
   context.subscriptions.push(vscode.commands.registerCommand('bitbake.open-recipe-workdir', async (uri) => { await openRecipeWorkdirCommand(bitbakeWorkspace, bitBakeProjectScanner, client, uri) }))
   context.subscriptions.push(vscode.commands.registerCommand('bitbake.recipe-devshell', async (uri) => { await openBitbakeDevshell(bitbakeTerminalProfileProvider, bitbakeWorkspace, bitBakeProjectScanner, uri) }))
   context.subscriptions.push(vscode.commands.registerCommand('bitbake.collapse-list', async () => { await collapseActiveList() }))
+  context.subscriptions.push(vscode.commands.registerCommand('bitbake.start-toaster-webview', async () => { ToasterPanel.createOrShow() }))
+  context.subscriptions.push(vscode.commands.registerCommand('bitbake.stop-toaster-webview', async () => { ToasterPanel.stop() }))
```

Comment:
Is there a reason to have a stop-webview when we can simply close the editor?

Reply:
Closing the editor only stops the webview but it doesn't stop the toaster.